opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -65,6 +65,8 @@
 import typing
 
 from opentelemetry import loader
+from opentelemetry import trace as trace_api
```

Comment:
This import can be removed and the `trace_api.SpanContext` below can be replaced with `'SpanContext'`! 😄

Reply:
Thanks! I'm fixing this...